Projekty/Projekt2/PatrykTomaszewskiMateuszZiemla/Milestone 1.ipynb

```diff
@@ -0,0 +1,1380 @@
+{
```

Comment:
<p>Dla rozkładów potęgowych można zobaczyć jak to wygląda w skali log, log - czyli logarytmujemy zarówno oś X, jak i Y. Wtedy zależność powinna być liniowa. To tak na przyszłość - nie ma konieczności teraz tego sprawdzać.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>